acir_tests/run_acir_tests.sh

```diff
@@ -7,7 +7,7 @@ set -e
 
 BB=$PWD/${BB:-../cpp/build/bin/bb}
 CRS_PATH=~/.bb-crs
-BRANCH=master
+BRANCH=acvm-0.21.0
```

Comment:
```suggestion
BRANCH=master
```

We should merge https://github.com/noir-lang/noir/pull/2051 before this PR and update this so that we track future changes in the test suite.